### PR TITLE
Add ssh key for Damian Nadales (dnadales).

### PR DESCRIPTION
**Pull request by dnadales**

### CODE REVIEW

**Comment by dnadales** [comment]
I also aligned the `=` signs in `csl-developers`, so it might be helpful to review this asking github to ignore whitespaces in the diff.

**Comment by dnadales** [comment]
@iohk-devops, I'm able to merge this pull request. I do not know if I should have this permission.

File: overlays/ssh-keys.nix [hunk 1]
```diff
@@ -30,11 +30,12 @@ lib: {
...
+    coot           = [ "ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEAs17FMCcqFt4GWaZdXilewWPveDFiS6NXuV5OKyosf6VkJLwu0mghEvxnMBdK4PyeUgalZzPRy8hb9JoGPkiD3SqZ0bOSUSY2NgUdUdbCDzuHdM3Ga/nSs2G/IvpY0c8Cyh1MCdYel4N2kQ3qWYkmBsfEScPXTV4DI/WhaSJzJBdhoz1NfNAZY+p6kmQ+6YIt2yziXFWT6p/JL0er+5KNd60ZE67n3crZ+X2fHNW/eUwL+VNUlSaejwRNAu0wgyvIiC+pdnET9NpvwcPooDU/FM5SkEWD4j1ivBUa0deCa0F5hmpEuok0oa2mac9+BN6gK3GmQ4fDKzrmpz2qsFQ0Cw==" ];
+    dnadales       = [ "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQC79R4WdKcB34PamcmHkEnafstXd8KUMhgreye8txe99LTs0HKwbSrDlIQ47zu6o4TdU8MbjynGRfEuApy8IBRO9swjj+K9X0vcyWIXgWwIfFVqGyqLcdbzF2PAHzS4TPHzIfWMlaT9cKZ4a6e60xwL6fJwfFyWzxD6vGF5KuVnOGEoHMvVao5TV/bcO1lJ3WagFdw0MpVoYjwM3km/QwZiAOAl28HlVTe+j6vN1Io6EaTnRMEXppB4rhQnBFCdB4xmkxCFMZwKwXuUEfP4C+ob6j49PCg5694Ig+7y4QOZo9Dc9YQU77hOYlDNPLdvgmjceCFDKuDcFWAnM5aatZGHtZCwZAYboF5n/eMQu6+IastJ6ns4f2abg3DQkykIflO56XQ2LtZW7y8pNAiO5ttkYA4KZbcUUO1G7RK/qzH+tgaY/iE5ySbIuOFI145W02p8DEsJu/EPzRJlO1GcpBCO15cXBpJzkUko2sAdomN59stj5bhCB7ucCIRWFOz4CE3uiQAR51R4A/ZIL2jcIHalJskWV99XJVq9YuWq8HrFGMeiK1FyL+q1ycHmmXbWv81MZ/Y5Upf21s8NWyWIotLf+6HWFzs8oCnNwS2Xb/NXps9Y1OZDblQyMpnwSNT7cmitRjU3SGr1UFdpAEW56maSM6JfFtdCKs+xTtKfPmZUKw== damian.only@gmail.com" ]
```
Comment (disassembler) [inline overlays/ssh-keys.nix:36]: missing `;`

**Review by disassembler — CHANGES_REQUESTED** [review]

**Comment by disassembler** [comment]
> @iohk-devops, I'm able to merge this pull request. I do not know if I should have this permission.

I've added CODEOWNERS and branch protection. Thanks for the heads up!

File: overlays/ssh-keys.nix (hunk 1 above)
Comment (disassembler) [reply overlays/ssh-keys.nix:36]:
```suggestion
    dnadales       = [ "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQC79R4WdKcB34PamcmHkEnafstXd8KUMhgreye8txe99LTs0HKwbSrDlIQ47zu6o4TdU8MbjynGRfEuApy8IBRO9swjj+K9X0vcyWIXgWwIfFVqGyqLcdbzF2PAHzS4TPHzIfWMlaT9cKZ4a6e60xwL6fJwfFyWzxD6vGF5KuVnOGEoHMvVao5TV/bcO1lJ3WagFdw0MpVoYjwM3km/QwZiAOAl28HlVTe+j6vN1Io6EaTnRMEXppB4rhQnBFCdB4xmkxCFMZwKwXuUEfP4C+ob6j49PCg5694Ig+7y4QOZo9Dc9YQU77hOYlDNPLdvgmjceCFDKuDcFWAnM5aatZGHtZCwZAYboF5n/eMQu6+IastJ6ns4f2abg3DQkykIflO56XQ2LtZW7y8pNAiO5ttkYA4KZbcUUO1G7RK/qzH+tgaY/iE5ySbIuOFI145W02p8DEsJu/EPzRJlO1GcpBCO15cXBpJzkUko2sAdomN59stj5bhCB7ucCIRWFOz4CE3uiQAR51R4A/ZIL2jcIHalJskWV99XJVq9YuWq8HrFGMeiK1FyL+q1ycHmmXbWv81MZ/Y5Upf21s8NWyWIotLf+6HWFzs8oCnNwS2Xb/NXps9Y1OZDblQyMpnwSNT7cmitRjU3SGr1UFdpAEW56maSM6JfFtdCKs+xTtKfPmZUKw== damian.only@gmail.com" ];
```

**Review by disassembler — CHANGES_REQUESTED** [review]

**Review by disassembler — APPROVED** [review]

**Pull request merged**